### DISCUSSION TITLE
There is no danish translations for vuetify controls!

### DISCUSSION THREAD

**Discussion by ruslanabdraev** [vuetifyjs/vuetify#14212] [General] [answered]
I found out that there are no danish translations for vuetify controls. See the following location - vuetify/packages/vuetify/src/locale/

Would it be possible to add danish translation?

**Comment by jacekkarczmarczyk** [comment, marked as answer]
There is already, not released yet
https://github.com/vuetifyjs/vuetify/pull/13733
